readme.md

```diff
@@ -20,9 +20,13 @@ This plugin adds support for the [Accelerated Mobile Pages](https://www.ampproje
 
```

Comment:
I think the description should lead with the benefits of the plugin. It can then describe the technical details. I'll also raise this internally, as we discussed.

Here are some possible first lines:

* Lightning-fast page loads, more native WordPress features, and no upsells. AMP, the WordPress way. ("no upsells" might be too negative)
* Bring the speed and features of the open-source [AMP](https://www.ampproject.org/) project to your site.

Reply:
Update: [this commit](https://github.com/Automattic/amp-wp/pull/1055/commits/03cd565e213fefd1c66cf506df64222243a37f1d) updates the leading line to:

>Bring the speed and features of the open source [AMP project](https://www.ampproject.org/) to your site, the WordPress way.